src/modules/api/transaction/dtos/index.ts

```diff
@@ -92,6 +92,7 @@ export class ViewPayoutStatusDto {
     @IsNumberString()
     limit: string;
 
+    @IsOptional()
     @IsEnum(TransactionStatus)
     payoutStatus: TransactionStatus;
```

Comment:
Why are you using this?

Reply:
I only changed the flag to optional, so I can use the (pagination)DTO for the new endpoint I'm creating.

Reply:
You are fetching recommended payouts only. This is not necessary